README.md

```diff
@@ -65,7 +65,7 @@ cd complex
 mkdir Release
 cd Release
 
-cmake -G Ninja -DCMAKE_TOOLCHAIN_FILE=C:/Applications/vcpkg/scripts/buildsystems/vcpkg.cmake -DCMAKE_BUILD_TYPE=Releas -DVCPKG_MANIFEST_DIR=C:/Users/mjackson/Workspace1/complex -DVCPKG_MANIFEST_FEATURES="tests;parallel" -DVCPKG_INSTALLED_DIR=C:/Users/mjackson/Workspace1/vcpkg-installed -DVCPKG_MANIFEST_INSTALL=ON -DCMAKE_MAKE_PROGRAM=C:/Applications/ninja-win/ninja.exe ../
+cmake -G Ninja -DCMAKE_TOOLCHAIN_FILE=/opt/local/vcpkg/scripts/buildsystems/vcpkg.cmake -DCMAKE_BUILD_TYPE=Release -DVCPKG_MANIFEST_FEATURES="tests;parallel" ../../complex
```

Comment:
This has a unix path instead of a Windows path.

Reply:
Fixed